starters/vue3-xstate-css/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <link rel="icon" href="/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
[meta element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta) is a [void element](https://developer.mozilla.org/en-US/docs/Glossary/Void_element) so it can't have a closing tag (and self closing as well)
I would also group meta tags for better readability
```suggestion
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="icon" href="/favicon.ico" />
```